README.md

```diff
@@ -1 +1,39 @@
-# ToDoList
\ No newline at end of file
+# ToDoList
+A To-do-ist javascript app buit using webpack and es6.
+
+Prerequisties
+HTML
+CSS
+JavaScript
+Wepack
+
+Getting Started
+Make sure you have node installed on your local machine.
+
+step 1: Clone this repo locally using git in the command line and the following command
+
+git clone https://github.com/asnganje/ToDoList.git
+
+Step 2: run npm install to download the necessary modules.
+
+step 3: run npm start to load the project in your broswer.
+
+
+Author  👤 Abdulrahman Nganje
+
+GitHub: https://github.com/asnganje
+
+LinkedIn: https://www.linkedin.com/in/abdulrahman-nganje-a6436935/
+
+🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support
+Give a ⭐️ if you like this project!
+
+Acknowledgments
+Miroverse Inc.
+📝 License
+This project is MIT licensed.
```

Comment:
- **[OPTIONAL]** I think it would be better if you improved the look and feel of your readme to be similar to that of the template. You can look at the raw version of the template to better understand how to edit markdown files. Here is the raw version: [https://raw.githubusercontent.com/microverseinc/readme-template/master/README.md](https://raw.githubusercontent.com/microverseinc/readme-template/master/README.md)

  ![readme-1](https://user-images.githubusercontent.com/54848363/184946033-a9f60e49-c9f5-4a8a-9114-93e819e225a7.jpg)